docs/api-reference/cloud-cli.md

```diff
@@ -179,12 +179,15 @@ This command unlinks a previously linked Postgres database instance.
 
 ### `npx dbos-cloud app register`
 
+:::info
+Deprecated, use `npx dbos-cloud app deploy` instead.
+:::
```

Comment:
Let's just remove the docs for this command?

Reply:
Not yet -- in case some users are still using the old version.

Reply:
Let's remove it--it's completely redundant now.